README.asciidoc

```diff
@@ -19,6 +19,7 @@ ShrinkWrap Resolvers is comprised of the following modules:
 |Maven Implementation with Archive Integration|org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-impl-maven-archive
 |Embedded Gradle API|org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-api-gradle-embedded-archive
 |Embedded Gradle Implementation with Archive Integration|org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-impl-gradle-embedded-archive
+|Implementation for Gradle Dependency Resolution|org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-impl-gradle
```

Comment:
No need to add that. There are already modules missing and I would rather prefer this section gone as part of the https://issues.redhat.com/browse/SHRINKRES-338